deploy/kubernetes/images/stable-1-21/image.yaml

```diff
@@ -1,53 +0,0 @@
-apiVersion: builtin
```

Comment:
Is there a reason why these files were removed? I think they are still in use. I think we could take a look at modifying these files in later commits.